src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -111,6 +127,61 @@ void function_call_harness_generatort::handle_option(
         "--" FUNCTION_HARNESS_GENERATOR_MAX_NONDET_TREE_DEPTH_OPT};
     }
   }
+  else if(option == FUNCTION_HARNESS_GENERATOR_MAX_ARRAY_SIZE_OPT)
+  {
+    p_impl->recursive_initialization_config.max_dynamic_array_size =
+      require_one_size_value(
+        FUNCTION_HARNESS_GENERATOR_MAX_ARRAY_SIZE_OPT, values);
+  }
+  else if(option == FUNCTION_HARNESS_GENERATOR_MIN_ARRAY_SIZE_OPT)
+  {
+    p_impl->recursive_initialization_config.min_dynamic_array_size =
+      require_one_size_value(
+        FUNCTION_HARNESS_GENERATOR_MIN_ARRAY_SIZE_OPT, values);
+  }
+  else if(option == FUNCTION_HARNESS_GENERATOR_TREAT_POINTER_AS_ARRAY_OPT)
+  {
+    std::copy(
+      values.begin(),
+      values.end(),
+      std::inserter(
+        p_impl->function_parameters_to_treat_as_arrays,
+        p_impl->function_parameters_to_treat_as_arrays.end()));
```

Comment:
⛏ `set` has a 2-argument insert method which tends to be more efficient and happens to be shorter.
```c++
p_impl->function_parameters_to_treat_as_arrays.insert(values.begin(), values.end());
```